html/pages/device/overview/processors.inc.php

```diff
@@ -16,44 +16,88 @@
     echo '</div>
         <table class="table table-hover table-condensed table-striped">';
 
+    $totalPercent=0;
+
     foreach ($processors as $proc) {
         $text_descr = rewrite_entity_descr($proc['processor_descr']);
 
         // disable short hrDeviceDescr. need to make this prettier.
         // $text_descr = short_hrDeviceDescr($proc['processor_descr']);
         $percent      = $proc['processor_usage'];
-        $background   = get_percentage_colours($percent);
-        $graph_colour = str_replace('#', '', $row_colour);
```

Comment:
Should this line and the above be moved? Are these variables not used in both parts of the if / else block?